R/dash.R

```diff
@@ -370,8 +377,32 @@ Dash <- R6::R6Class(
     # private fields defined on initiation
     name = NULL,
     serve_locally = NULL,
-    routes_pathname_prefix = NULL,
-    requests_pathname_prefix = NULL,
+    routes_pathname_prefix = {
+      routes_prefix <- Sys.getenv("DASH_ROUTES_PATHNAME_PREFIX")
```

Comment:
These variables are also defined in the constructor: 
```
routes_pathname_prefix = '/',
requests_pathname_prefix = '/',
```

The order of preference should be:
1. Did the user supply it in the constructor? If so, then use that.
2. If not, then check the environment
3. If not in the environment, then use the default.

In Python, for (1), the only way to check if a user supplied it while keeping a default argument was to set that default argument to `None`. That way, if it's not `None`, then we can assume that the user set it. (If the default was `/`, then we wouldn't know whether the user explicitly supplied that or if its just the default).

Reply:
OK, perfect. This makes sense. I’ll just use `NULL` instead of `None`.

Reply:
fixed in 5fba10f16601ebe26b15f3b96334105b7eafff31